.circleci/test-deploy.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           paths:
             - .cache/Cypress
             - project
-  run-tests-in-chrome:
+  run-ct-tests-in-chrome:
```

Comment:
I think we should consider renaming the `examples/angular-app` name to include CT somehow if this is only running & showing examples for CT tests.